src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -548,11 +571,26 @@ public function overrideGlobals()
      *
      * You should only list the reverse proxies that you manage directly.
      *
-     * @param array $proxies A list of trusted proxies
+     * @param array $proxies          A list of trusted proxies
+     * @param int   $trustedHeaderSet A bit field of Request::HEADER_*, usually either Request::HEADER_FORWARDED or Request::HEADER_X_FORWARDED_ALL, to set which headers to trust from your proxies
```

Comment:
I means that you can trust the `Forwarded` and the `X-Forwarded-For` headers at the same time. I don't think that this is desirable.